CMP1903_A1_2324/Game.cs

```diff
@@ -8,14 +8,32 @@ namespace CMP1903_A1_2324
 {
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+        public Die[] Dice { get; private set; } = new Die[3];
 
-        //Methods
+        public Game()
+        {
+            for (int i = 0; i < 3; i++)
+            {
+                Dice[i] = new Die();
+            }
+        }
 
+        public int DiceRoll()
+        {
+            int sum = 0;
+
+            for (int i = 0; i < 3; i++)
+            {
+                int roll = Dice[i].Roll();
+                Console.WriteLine($"Die {i + 1} rolled: {roll}");
+                sum += roll;
+            }
+
+            Console.WriteLine("--------------------------------");
+            Console.WriteLine($"Sum of the rolled dice is: {sum}");
+
+            return sum;
+        }
     }
+
 }
```

Comment:
The code follows C# style conventions. Functions well.

Reply:
Should add comments for ease of reading